Dockerfile

```diff
@@ -1,16 +1,12 @@
 # basic python3 image as base
-FROM continuumio/miniconda3
+FROM continuumio/miniconda3:22.11.1-alpine
```

Comment:
Is there a reason to pin this? If not, I'd prefer not to, because we need to maintain it then. If we need to pin it, could you add a comment why? That way, we know why in the future and don't remove it by mistake.

Reply:
Fixed it. The alpine one was old anyway